### PR TITLE
Support yelp-cheetah 0.16.x

### PR DESCRIPTION
**Pull request by asottile**
I need to make a release so the test'll pass on travis, but they pass locally with current master:

```
$ coverage erase && coverage run -m pytest tests &&  coverage report --show-missing --skip-covered --omit=venv*
============================================================================================ test session starts ============================================================================================
platform linux2 -- Python 2.7.6, pytest-2.9.2, py-1.4.31, pluggy-0.3.1
rootdir: /home/asottile/workspace/refactorlib, inifile: 
collected 118 items 

tests/cheetah/add_comment_test.py .
tests/cheetah/directive_test.py .........
tests/cheetah/is_in_context_test.py ..
tests/cheetah/multiline_test.py ..
tests/cheetah/parse_test.py ....................................
tests/cheetah/remove_call_test.py .....
tests/cheetah/remove_decorator_test.py ........
tests/javascript/parse_test.py ...........................
tests/python/parse_test.py ............................

======================================================================================== 118 passed in 4.18 seconds =========================================================================================
Name                                      Stmts   Miss Branch BrPart  Cover   Missing
-------------------------------------------------------------------------------------
refactorlib/__init__.py                       9      1      2      1    82%   7, 5->7
refactorlib/cheetah/node.py                 170     21     40      3    89%   79-80, 103, 115-132, 142, 216-217, 12->14, 99->103, 104->98
refactorlib/cheetah/parse.py                146     11     60      5    91%   141-144, 170-172, 222, 228, 230, 232, 72->76, 162->170, 227->228, 229->230, 231->232
refactorlib/cli/xmlfrom.py                   12      3      0      0    75%   19, 24-25
refactorlib/cli/xmlstrip.py                  11      3      0      0    73%   18, 23-24
refactorlib/dictnode.py                      25      1      6      0    97%   8
refactorlib/filetypes.py                     42      7     14      3    82%   8, 47, 53, 56-59, 7->8, 46->47, 49->53
refactorlib/javascript/parse.py              60      4     24      4    90%   30, 79, 86, 93, 29->30, 76->79, 85->86, 89->93
refactorlib/node.py                          49      7     10      3    83%   39, 52-55, 72, 84, 25->27, 30->32, 66->72
refactorlib/parse.py                         32      0     10      1    98%   12->15
refactorlib/python/parse.py                  62      0     18      1    99%   97->102
refactorlib/util.py                          24      3      6      2    83%   20, 48-49, 17->20, 47->48
testing/util.py                              76     13     26      5    82%   30, 33, 41, 82-87, 101-104, 118, 28->30, 31->33, 40->41, 100->101, 117->118
-------------------------------------------------------------------------------------
TOTAL                                       926     74    248     28    91%

22 files skipped due to complete coverage.
```

### CODE REVIEW

File: tests/cheetah/remove_call_data/test_can_remove_calls/multiline.tmpl [hunk 1]
```diff
@@ -1,7 +1,5 @@
         foo  bar
-        foo $dotted.name bar
+        foo ${dotted.name} bar
         foo $cheetahvar bar
         foo $cheetah.var bar
-        foo $(
-            x.upper() for x in mylist
-        ) bar
```
Comment (bukzor) [inline tests/cheetah/remove_call_data/test_can_remove_calls/multiline.tmpl]: I believe the point of this testcase was the multiline expression.
Isn't that valid anymore?

File: tests/cheetah/remove_call_test.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ def test_can_remove_calls(example, output):
     example = parse(example)
 
     calls = example.find_calls('foo')
-    assert len(calls) == 5
```
Comment (bukzor) [inline tests/cheetah/remove_call_test.py]: why change?

File: refactorlib/cheetah/node.py [hunk 3]
```diff
@@ -121,115 +113,93 @@ def call(self, method, arguments):
...
         return self.args_container.getchildren()
 
     def remove_call(self):
-        args_body = self.args_body
-        args_container = one(args_body.xpath('./CallArgString'))
         args = self.args
+        assert args[0].tag == 'Py' and args[0].text == '('
+        assert args[-1].tag == 'Py' and args[-1].text == ')'
+        args = args[1:-1]
+        args = [arg for arg in args if arg.text.strip() or arg.tail.strip()]
```
Comment (bukzor) [inline refactorlib/cheetah/node.py]: I think this logic is meant to live under `.args`.

It doesn't seem right if .args has things that are not arguments.
I think that's the distinction between "args" and "args container" that you see on LHS.

**Comment by bukzor** [comment]
nits.

shipit.

File: tests/cheetah/remove_call_test.py (hunk 2 above)
Comment (asottile) [inline tests/cheetah/remove_call_test.py]: I added more calls to one of the test inputs.  This was asserting that each of the input files had exactly 5 `$foo` calls, which seemed a little silly to me

File: tests/cheetah/remove_call_data/test_can_remove_calls/multiline.tmpl (hunk 1 above)
Comment (asottile) [inline tests/cheetah/remove_call_data/test_can_remove_calls/multiline.tmpl]: this is the output -- the input is still multilined.  I made the argument thingy more clever with non-important whitespace

**Pull request merged**